src/pages/debrief/debrief.html

```diff
@@ -19,82 +19,97 @@ <h1>unsuccessful</h1>
 
 <ion-content>
 
-  <ion-card id="dangerous-fault" *ngIf="(pageState.dangerousFaults$ | async).length > 0">
-    <ion-card-header>
-      <ion-card-title>
-        <h1 class="fault-heading">{{(pageState.dangerousFaults$ | async).length}}</h1> <h4 class="fault-heading">Dangerous</h4>
-      </ion-card-title>
-    </ion-card-header>
+  <ion-card id="ETA" *ngIf="(pageState.etaVerbal$ | async) || (pageState.etaPhysical$ | async)">
     <ion-card-content>
-        <ion-grid>
-            <ion-row align-items-center *ngFor="let dangerousFault of (pageState.dangerousFaults$ | async)" class="mes-data">
-              <ion-col col-32></ion-col>
-              <ion-col col-2>
-              <div class="counter-icon">
-                <dangerous-fault-badge [showBadge]="true"></dangerous-fault-badge>
-              </div>
-            </ion-col>
-             <ion-col col-3></ion-col>
-              <ion-col><div class="counter-label">{{dangerousFault}}</div></ion-col>
-            </ion-row>
-          </ion-grid>
+      <ion-grid>
+        <ion-row align-items-start class="mes-data">
+          <ion-col col-3></ion-col>
+          <ion-col col-34>
+            <h4 class="fault-heading">ETA</h4>
+          </ion-col>
+          <ion-col align-self-start>
+            <div id="physical" *ngIf="!(pageState.etaVerbal$ | async) && (pageState.etaPhysical$ | async)">Physical</div>
```

Comment:
Instead of having 3 separate divs for this, could you not create a single Observable of the ETA text which is derived from the latest values of ETA physical + ETA verbal?

Reply:
done